### PR TITLE
Fix noisy Rubocop complaints

### PR DESCRIPTION
**Pull request by rpunt**
## Description
Quite noisy Rubocop complaints about things like double-quote non-interpolated strings, square-bracket-delimited arrays, and the like.

## Motivation and Context
My dev environment run rubocop and foodcritic as a pre-commit hook; to ingest vendor changes, these fixes are required.

## How Has This Been Tested?
Converged when encapsulated in a wrapper cookbook.

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Linter fixes

## Checklist:
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

File: recipes/_linux.rb [hunk 1]
```diff
@@ -40,55 +40,59 @@
...
+        node["platform_version"],
+        node['sensu']['version_suffix']
+      )
+    }
     allow_downgrade true
     notifies :create, "ruby_block[sensu_service_trigger]"
   end
 else
-  raise "Unsupported Linux platform family #{platform_family}"
+  fail "Unsupported Linux platform family #{platform_family}"
```
[the remark this answers is not in the document]
Comment (majormoses) [reply recipes/_linux.rb]: Looks like maybe we just need to update rubocop: https://github.com/bbatsov/rubocop/issues/2732#issuecomment-176416811

**Comment by majormoses** [comment]
@rpunt I rebased your fork to fix some merge conflicts and squashed. I also fixed up a few more things mostly around single quotes when string interpolation was not needed but a few minor things that were bugging me. @thomasriley I could use a second pair of eyes on this one, it's long and tedious review and I added stuff to it so I minimally need someone to review those changes I made.

**Comment by epierotto** [comment]
Usually [cookstyle](https://docs.chef.io/cookstyle.html) is used as linting tool for developing Chef cookbooks, it's based on Rubocop but is specific for this task.

**Comment by majormoses** [comment]
@epierotto ya `cookstyle` has only been around for a few years and this cookbook pre-dates that by quite a bit. Also I personally found little value in it over just releasing a chef specific `.rubocop.yml` to use as a guideline as it nice to have your editor show you when you start to deviate from best practices and I touch a lot of non chef ruby so when I have to choose I choose the more generic one and customize it. I am not opposed to moving to it if someone wants to put in the effort but I need a compelling reason to put in the effort personally.

**Comment by majormoses** [comment]
Due to the lack of activity and conflicts I am gonna close this. Going forward we will be using cookstyle.

**Pull request closed without merging**